data-model/fdo-types/0.3.0/shared-models/schema/agent.json

```diff
@@ -0,0 +1,80 @@
+{
+  "$id": "https://schemas.dissco.tech/schemas/fdo-types/0.3.0/shared-models/agent.json",
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "type": "object",
+  "properties": {
+    "ods:agentRole": {
+      "type": "string",
+      "description": "Indicates the role of the agent",
+      "examples": [
+        "collector",
+        "preparer",
+        "identifier",
+        "recorder"
+      ]
+    },
+    "ods:agentType": {
+      "type": "string",
+      "description": "Indicates the type of agent",
+      "examples": [
+        "machine",
+        "human",
+        "organisation"
+      ]
+    },
+    "ods:agentID":{
+      "type": "string",
+      "description": "Primary identifier of the agent, additional identifiers can go in the identifiers array",
+      "examples": [
+        "https://orcid.org/0000-0002-1825-0097"
+      ]
+    },
+    "ods:agentName": {
+      "type": "string",
+      "description": "Full name of the agent",
+      "examples": [
+        "John Smith"
+      ]
+    },
+    "ods:agentRoleBegan": {
+      "type": "string",
+      "description": "Date the agent began the role",
+      "examples": [
+        "2023-10-02T12:31:34.806Z"
+      ]
+    },
+    "ods:agentRoleEnded": {
+      "type": "string",
+      "description": "Date the agent ended the role",
+      "examples": [
+        "2023-09-02T12:31:34.806Z"
+      ]
+    },
+    "ods:agentRoleOrder": {
+      "type": "integer",
+      "description": "Order of the agent in the role. Can be used to indicate the order of importance",
+      "minimum": 1,
+      "examples": [
+        1,
+        2
+      ]
+    },
```

Comment:
Can you explain a situation where this could be applied to?

Reply:
Yes when you have a primary author and a secondary, so if two agents have the same role but there is hierarchy in the relationship. Not sure when we will use it but took it over from I think GBIF UM which took it from the TDWG group